x-pack/platform/plugins/shared/index_management/public/application/index.tsx

```diff
@@ -96,7 +96,7 @@ export const IndexManagementAppContext: React.FC<IndexManagementAppContextProps>
 };
 
 export const renderApp = (
-  elem: HTMLElement | null,
+  elem: HTMLDivElement | HTMLElement | null,
```

Comment:
Is this needed? I would think that `HTMLDivElement` inherits `HTMLElement`

Reply:
updated in https://github.com/elastic/kibana/commit/3343d3721544521ec0f49a2486e904e29903dd04